app/decorators/guest_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class GuestDecorator
+  def lockout_time_remaining_in_words
```

Comment:
One way would be to set a default type to `generic` and have a `max_generic_login_attempts_reached` i18n entry, or something like that.